scripts/docker.env

```diff
@@ -2,4 +2,5 @@
 # - `-Xregexp.interruptible=true -Xcompile.invokedynamic=true -Xjit.threshold=0` LS base-line
 # - `-XX:+UseParallelGC` do not use G1 (default) on Java 11
 # - `-v -W1` print JRuby version but do not go verbose
-JRUBY_OPTS=-Xregexp.interruptible=true -Xcompile.invokedynamic=true -Xjit.threshold=0 -J-XX:+UseParallelGC -J-XX:+PrintCommandLineFlags -v -W1
\ No newline at end of file
+JRUBY_OPTS=-Xregexp.interruptible=true -Xcompile.invokedynamic=true -Xjit.threshold=0 -J-XX:+UseParallelGC -J-XX:+PrintCommandLineFlags -v -W1
+OPENSEARCH_INITIAL_ADMIN_PASSWORD=myStrongPassword123!
```

Comment:
If we change this to `admin`, then we can use `admin` below and it should work for older versions of OpenSearch. Can we make that change?

Reply:
Also, `admin` is recognized as weak password and so the setup would fail. Hence we defined a new password with relatively strong value `myStrongPassword123!`